src/nativeMain/kotlin/com/autodesk/coroutineworker/WorkerPool.kt

```diff
@@ -19,7 +18,6 @@ private class WeightedWorker(
     val numBlocksQueued = AtomicLong(0)
 
     companion object {
-        @SharedImmutable
```

Comment:
Why is the shared immutable thing going away?

Reply:
It's only required for top-level globals. object vals are already frozen